sdk/dotnet/Glacier/Vault.cs

```diff
@@ -230,17 +221,11 @@ public sealed class VaultState : Pulumi.ResourceArgs
         [Input("name")]
         public Input<string>? Name { get; set; }
 
-        [Input("notifications")]
-        private InputList<Inputs.VaultNotificationGetArgs>? _notifications;
-
         /// <summary>
         /// The notifications for the Vault. Fields documented below.
         /// </summary>
-        public InputList<Inputs.VaultNotificationGetArgs> Notifications
-        {
-            get => _notifications ?? (_notifications = new InputList<Inputs.VaultNotificationGetArgs>());
-            set => _notifications = value;
-        }
```

Comment:
Breaking changes in this resource... let's triage

Reply:
Fixed!

Reply:
`Notifications` changed to `Notification`